### PR TITLE
switch to dedicated consumption plan

### PR DESCRIPTION
**Pull request by SandraAhlgrimm**
I also updated the tracking links to the latest format

### CODE REVIEW

**Review by mkheck — CHANGES_REQUESTED** [review]
Lines 47, 52, and 60 in 01-create.../README.md have an extraneous "env" in the command:

Line 47: --name $AZ_SPRING_APPS_NAME env \

Should be:

--name $AZ_SPRING_APPS_NAME \

Same for lines 52 & 60.

**Comment by SandraAhlgrimm** [comment]
> Lines 47, 52, and 60 in 01-create.../README.md have an extraneous "env" in the command:
> 
> Line 47: --name $AZ_SPRING_APPS_NAME env \
> 
> Should be:
> 
> --name $AZ_SPRING_APPS_NAME \
> 
> Same for lines 52 & 60.

Yeah it’s the name for the container environment, that’s why I used a concat string with the spring apps instance name here

**Comment by mkheck** [comment]
> > Lines 47, 52, and 60 in 01-create.../README.md have an extraneous "env" in the command:
> > Line 47: --name $AZ_SPRING_APPS_NAME env \
> > Should be:
> > --name $AZ_SPRING_APPS_NAME \
> > Same for lines 52 & 60.
> 
> Yeah it’s the name for the container environment, that’s why I used a concat string with the spring apps instance name here

If that's the case, there shouldn't be a space between the $vbl and the 'env', correct?

Should be something like $AZ_SPRING_APPS_NAME'env' or $AZ_SPRING_APPS_NAME'-env', yes?

**Comment by SandraAhlgrimm** [comment]
> > > Lines 47, 52, and 60 in 01-create.../README.md have an extraneous "env" in the command:
> > > Line 47: --name $AZ_SPRING_APPS_NAME env 
> > > Should be:
> > > --name $AZ_SPRING_APPS_NAME 
> > > Same for lines 52 & 60.
> > 
> > 
> > Yeah it’s the name for the container environment, that’s why I used a concat string with the spring apps instance name here
> 
> If that's the case, there shouldn't be a space between the $vbl and the 'env', correct?
> 
> Should be something like $AZ_SPRING_APPS_NAME'env' or $AZ_SPRING_APPS_NAME'-env', yes?

Or should we export a new variable for this? I would still want the variable name to be generated as I see no benefit for the user to specify a specific name here.

**Comment by mkheck** [comment]
> > > > Lines 47, 52, and 60 in 01-create.../README.md have an extraneous "env" in the command:
> > > > Line 47: --name $AZ_SPRING_APPS_NAME env
> > > > Should be:
> > > > --name $AZ_SPRING_APPS_NAME
> > > > Same for lines 52 & 60.
> > > 
> > > 
> > > Yeah it’s the name for the container environment, that’s why I used a concat string with the spring apps instance name here
> > 
> > 
> > If that's the case, there shouldn't be a space between the $vbl and the 'env', correct?
> > Should be something like $AZ_SPRING_APPS_NAME'env' or $AZ_SPRING_APPS_NAME'-env', yes?
> 
> Or should we export a new variable for this? I would still want the variable name to be generated as I see no benefit for the user to specify a specific name here.

I would _greatly_ prefer to export a purpose-specific variable for this...but as long as we're consistent, I would be happy. Let me test something and place an updated suggestion here shortly.

**Comment by mkheck** [comment]
Suggestion:

Add another env var similar to the following to the initial variable config:

export AZ_CONTAINER_APP_ENV=$AZ_SPRING_APPS_NAME'-container'

Then replace the lines called out above with:

--name $AZ_CONTAINER_APP_ENV

as applicable.

Seems the cleanest way forward, and I've confirmed it completes successfully.

File: 01-create-an-azure-spring-cloud-instance/README.md [hunk 1]
```diff
@@ -38,13 +38,38 @@ export AZ_RESOURCE_GROUP=spring-apps-lab
...
+    --enable-workload-profiles
+
+az containerapp env workload-profile set \
+    --resource-group $AZ_RESOURCE_GROUP \
+    --name $AZ_SPRING_APPS_NAME env \
+    --workload-profile-name my-wlp \
+    --workload-profile-type D4 \
+    --min-nodes 1 \
+    --max-nodes 2
+
```
Comment (SandraAhlgrimm) [inline 01-create-an-azure-spring-cloud-instance/README.md:57]: `export AZ_CONTAINER_APP_ENV=$AZ_SPRING_APPS_NAME'-container'`

**Review by mkheck — APPROVED** [review]

**Pull request merged**